src/main/resources/swagger/ga4gh-tool-discovery.yaml

```diff
@@ -599,65 +616,41 @@ definitions:
     type: string
     description: >-
       The type of descriptor that represents this version of the tool (e.g. CWL,
-      WDL, or NFL).
+      WDL, or NFL). Note that these files can also include associated Docker/container files 
+      and test parameters that further describe a version of a tool
     enum:
       - CWL
       - WDL
       - NFL
-  ToolDescriptor:
+  FileWrapper:
     type: object
-    description: A tool descriptor is a metadata document that describes one or more tools.
+    description: A file provides content for one of; 
+      - A tool descriptor is a metadata document that describes one or more tools.
+      - A tool document that describes how to test with one or more sample test
+      JSON.
+      - A containerfile is a document that describes how to build a particular
+      container image. Examples include Dockerfiles for creating Docker images
+      and Singularity recipes for Singularity images
     required:
       - type
     properties:
       type:
         $ref: '#/definitions/DescriptorType'
       descriptor:
         type: string
-        description: The descriptor that represents this version of the tool.
+        description: The content of the file itself. One of url or descriptor is required.
```

Comment:
should this property be renamed "content"?

Reply:
I was considering it, but it would be a larger breaking change for DNANexus. Thoughts @coverbeck ?

Reply:
`content` seems  right. It seems like we should work with DNAnexus to get through this.